.php_cs.dist

```diff
@@ -6,18 +6,13 @@ if (!file_exists(__DIR__.'/src')) {
 
 return PhpCsFixer\Config::create()
     ->setRules([
+        '@PHP71Migration' => true,
+        '@PHPUnit75Migration:risky' => true,        
         '@Symfony' => true,
         '@Symfony:risky' => true,
-        '@PHPUnit75Migration:risky' => true,
-        'php_unit_dedicate_assert' => ['target' => '5.6'],
-        'array_syntax' => ['syntax' => 'short'],
-        'fopen_flags' => false,
         'protected_to_private' => false,
         'native_constant_invocation' => true,
-        'combine_nested_dirname' => true,
         'list_syntax' => ['syntax' => 'short'],
```

Comment:
FYI we gonna incorporate it in PHP migration ruleset https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5428

Reply:
then let's remove this line too, and merge this PR after FriendsOfPHP/PHP-CS-Fixer#5428 is merged?

Reply:
we need to extend integration tests to reflect this change. can't promise it's done in few days.
that's why i decided to have PR to cleanup what's currently possible, and do further cleanup in future, when changes on PHP CS Fixer are done and when fabbot is updated